solutions/devsprint-bruno-faganello-2/SpaceApp/Utils/Extensions/String/SPStringExtension.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  SPStringExtension.swift
+//  SpaceApp
+//
+//  Created by Paolo Prodossimo Lopes on 25/04/22.
+//
+
+import SwiftUI
+
+struct SPStringExtension {
```

Comment:
[QUESTION]
Why this?

Reply:
I usualy use this struct to give me some custom methods and properties, especifics about project. I think this looks more didactic during development, this class is accessible throws especific propertie in extension.

Reply:
done